### ISSUE TITLE
AllGroupProjectsOptions is missing archived parameter

### ISSUE THREAD

**Issue by r1m** [jdalrymple/gitbeaker#3406]
**Description**

- Node.js version: 18
- Gitbeaker version: 9.13.0
- Gitbeaker release (cli, node, browser, core, requester-utils): rest
- OS & version: ubuntu
- Gitlab version: 16

https://archives.docs.gitlab.com/14.10/ee/api/groups.html#list-a-groups-projects
lists `archived` as a way to filter projects but this is not available in AllGroupProjectsOptions 

Attribute | Type | Required | Description
-- | -- | -- | --
archived | boolean | no | Limit by archived status

**Steps to reproduce**

```ts
 const projects = await this.client.Groups.allProjects(groupId,{withShared:false, archived:false});

//  Object literal may only specify known properties, and 'archived' does not exist in type 'AllGroupProjectsOptions & BasePaginationRequestOptions<"offset">
```

**Checklist**
- [x] I have checked that this is not a duplicate issue.
- [x] I have read the documentation.

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`39.14.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/39.14.0) :rocket:

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`39.15.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/39.15.0) :rocket:
